components/lists/FeedList.tsx

```diff
@@ -8,8 +16,90 @@ import { View } from 'react-native';
  * - add tests for the feed list component
  */
 
+interface FeedListItemProps {
+  item: Feed;
+}
+
+const { height } = Dimensions.get('screen');
+
+const FeedListItem = ({ item }: FeedListItemProps) => {
+  const { colors } = useTheme();
+  return (
+    <View style={styles.feed} testID="feedItem">
+      <StyledText size="md">{item.body}</StyledText>
```

Comment:
Please add font "regular" prop to styled text so that it matches the design's text styling

Reply:
@shyrynaz also incase you have inquiries or need assistance, you can contact any of us. We even have a telegram group where you can just interact with any of us.https://t.me/+bJGp4hqa4i40NTY0